src/components/inputs/Search/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,21 @@
+import { ReactComponent as SearchIcon } from '../../../icons/Search.svg';
+import './styles.css';
+import { searchBar } from '../../../../types';
+
+function SearchBar({ searchButtonAction }: searchBar) {
+  return (
+    <div className="searchBar">
+      <input type="text" placeholder="sam" />
+      <button
+        className="searchButton"
+        onClick={() => {
+          searchButtonAction();
+        }}
+      >
+        <SearchIcon fill="black" stroke="white" width="18.51" />
+      </button>
```

Comment:
per figma - the placeholder font size in the search bar looks heavy/bold and could use additional padding. also, the placeholder should likely be a prop passed into the component 
<img width="1603" alt="image" src="https://user-images.githubusercontent.com/12800001/199808965-23db1e4b-9ebb-4937-b239-21fbd59fb4e4.png">
vs. 
<img width="755" alt="image" src="https://user-images.githubusercontent.com/12800001/199809253-45a62ef1-1b3a-4011-afc8-dd839c993990.png">

Reply:
as well is this in the console on load:
```
Error: <svg> attribute height: Expected length, "current".
setValueForProperty @ react-dom.development.js:798

```